### PR TITLE
forc wallet mvp

### PR DESCRIPTION
**Pull request by Voxelot**
This mvp was originally created with the intention of basic bip39 based keygen, with helpers to render the keys into the native address format used by fuel. I wasn't aware of the original design in https://github.com/FuelLabs/forc-wallet/issues/2, so if this is too rudimentary for the wallet we could potentially make a separate dumbed down cli called `forc-keygen` and move this functionality over there.

Try it out for yourself!

```
git clone -b mvp git@github.com:FuelLabs/forc-wallet.git
cd forc-wallet
cargo run -- --help
```

Generates wallets into json or toml format:
```
cargo run -- generate

{
  "mnemonic": "antique draft sudden frown order entry time portion price equip novel second",
  "secret": "0x0f9966304c291f463765655096edeb5bbc9cd72bc7eb283e6e309a3051224861",
  "address": "0x9014317b320caf75e20528a7faea2c2b5ca4df41938ff08ba2f189d465343246"
}
```

```
cargo run -- -o toml generate

mnemonic = "lemon trade gentle adult resist census virtual wheel tape measure twin endorse"
secret = "0xbee5e1fc218ced71a74e60eaf554ea8ce70bb8a8f6697e3fb8b3de5ec02cfb01"
address = "0x55def963d41bea46f533b6ae1b3c54236f2790c7becd839a9faa244e4da4e48f"
```

### CODE REVIEW

**Review by digorithm — APPROVED** [review]
LGTM -- it should be super easy to integrate with the SDK once the SDK supports mnemonics and encrypted files! ETA ~2 days.

Tracking it [here](https://github.com/FuelLabs/fuels-rs/issues/126) and [here](https://github.com/FuelLabs/fuels-rs/issues/127).

File: README.md [hunk 1]
```diff
@@ -2,3 +2,24 @@ forc-wallet
 ===
 
 A forc plugin for generating or importing wallets using BIP39 phrases.
+
+## Instructions
+
+```shell
+cargo run -- --help
+```
+```
```
Comment (adlerjohn) [inline README.md:11]:
````suggestion
```

```
````

File: README.md [hunk 2]
```diff
@@ -2,3 +2,24 @@ forc-wallet
...
+    -h, --help            Print help information
+    -o, --format <FMT>    [default: json] [possible values: json, toml]
+    -V, --version         Print version information
+
+SUBCOMMANDS:
+    generate    Randomly generate a new wallet
+    help        Print this message or the help of the given subcommand(s)
+    import      Import a wallet from mnemonic phrase
+
+```
```
Comment (adlerjohn) [inline README.md:25]:
````suggestion
    import      Import a wallet from mnemonic phrase
```
````

**Review by adlerjohn — APPROVED** [review]

**Comment by mitchmindtree** [comment]
I'm a bit slow on reviewing this but just wanna say this is such a nice MVP/start - so concise and really cuts to the essentials!

**Pull request merged**